certbot/certbot/util.py

```diff
@@ -567,6 +560,21 @@ def enforce_domain_sanity(domain):
     return domain
 
 
+def is_ipaddress(address):
```

Comment:
Isn't this redundant to `_is_ip(address)`? Is that to avoid dependency on `crypto_util.py`?

Reply:
because configuration didn't import acme into it, I avoided to make it new dependency. btw now I looking at it if tills better expose _is_ip(removeing first undercore), but it looks like not really in scope of crypto_util. maybe expose it and move it to util.py?

Reply:
I like the idea of avoiding redundant code. `util.py` sounds to be good home for this method. 
@bmw might be able to weigh in on this one.